app/pages/amplience/page/index.jsx

```diff
@@ -0,0 +1,143 @@
+import {Box, Heading, Skeleton} from '@chakra-ui/react'
+
+// Project Components
+import Seo from '../../../components/amplience/seo'
+
+// Amplience Wrapper Component
+import AmplienceWrapper from '../../../components/amplience/Wrapper'
+import {RealtimeVisualization} from '../../../contexts'
+
+// Constants
+import {MAX_CACHE_AGE} from '../../../constants'
+
+/**
+ * This is an example content page for Retail React App.
+ * The page is created for demonstration purposes.
+ * The page renders SEO metadata and a few promotion
+ * categories and products, data is from local file.
+ */
+const ContentPage = ({page}) => {
+    //const intl = useIntl()
+
+    const RTV = useContext(RealtimeVisualization)
+    let removeChangedSubscription = undefined
+    const [pageModel, setPageModel] = useState(undefined)
+
+    useEffect(() => {
+        setPageModel(page)
+        console.log('page', pageModel)
+    }, [page])
+
+    useEffect(() => {
+        if (RTV.ampVizSdk !== null) {
+            RTV.ampVizSdk.form.saved(() => {
+                window.location.reload()
+            })
+
+            removeChangedSubscription = RTV.ampVizSdk.form.changed((model) => {
+                // handle form model change
+                setPageModel(model.content)
+            })
+        }
+
+        return () => {
+            if (removeChangedSubscription != undefined) {
+                removeChangedSubscription()
+            }
+        }
+    }, [RTV.ampVizSdk])
+
+    return (
+        <Box data-testid="amplience-page" layerStyle="page">
+            {pageModel == undefined ? (
+                <Skeleton height="20px" />
+            ) : (
+                <>
+                    <Seo
+                        title={pageModel.seo.title}
+                        description={pageModel.seo.description}
+                        keywords={pageModel.seo.keywords}
+                        noIndex={pageModel.seo.noindex}
+                    />
+                    <Heading
+                        as="h1"
+                        fontSize={{base: '4xl', md: '5xl', lg: '6xl'}}
+                        maxWidth={{base: '75%', md: '50%', lg: 'md'}}
+                    >
+                        {pageModel.seo.title}
+                    </Heading>
+                </>
+            )}
+            {pageModel == undefined ? (
+                <Skeleton height="200px" />
+            ) : (
+                <>
+                    {pageModel.content.map((item) => {
+                        return <AmplienceWrapper key={item._meta.deliveryId} content={item} />
+                    })}
+                </>
+            )}
+        </Box>
+    )
+}
+
+ContentPage.getTemplateName = () => 'contentpage'
+
+ContentPage.shouldGetProps = ({previousLocation, location}) =>
+    !previousLocation || previousLocation.pathname !== location.pathname
+
+ContentPage.getProps = async ({res, params, location, api, ampClient}) => {
+    const {pageId} = params
+
+    if (res) {
```

Comment:
On the homepage, this is not set if ampClient has a vse.

Reply:
updated to (res && !ampClient.vse)